tests/functional/emit-schema-sdl.ts

```diff
@@ -52,126 +42,235 @@ describe("Emitting schema definition file", () => {
         return {} as MyObject;
       }
     }
+
     MyResolverClass = MyResolver;
 
     schema = await buildSchema({
       resolvers: [MyResolver],
     });
   });
 
-  function checkSchemaSDL(commentPrefix: "#" | `"""` = `"""`) {
-    expect(contentArg).toContain("THIS FILE WAS GENERATED");
-    expect(contentArg).toContain("MyObject");
+  function checkSchemaSDL(contents: string, commentPrefix = `"""`) {
+    expect(contents).toContain("THIS FILE WAS GENERATED");
+    expect(contents).toContain("MyObject");
     if (commentPrefix === `"""`) {
-      expect(contentArg).toContain(`"""Description test"""`);
+      expect(contents).toContain(`"""Description test"""`);
     } else {
-      expect(contentArg).toContain(`# Description test`);
+      expect(contents).toContain(`# Description test`);
     }
   }
 
   describe("emitSchemaDefinitionFile", () => {
     it("should call writing file with schema SDL", async () => {
-      await emitSchemaDefinitionFile("testPath1", schema);
-      expect(pathArg).toEqual("testPath1");
-      checkSchemaSDL();
+      const targetPath = path.join(TEST_DIR, "schemas", "test1", "schema.gql");
+      await emitSchemaDefinitionFile(targetPath, schema);
+      expect(fs.existsSync(targetPath)).toEqual(true);
+      checkSchemaSDL(fs.readFileSync(targetPath).toString());
+    });
+
+    it("should call writing file with schema SDL", async () => {
+      const targetPath = "./type-graphql/schemas/schema.gql";
```

Comment:
Looks like you have a lot of duplicated test - is this a merge error or you write the path by hand on purpose?

Reply:
It was by design. As I want to be sure if users provide simple string or from `path.resolve`.

Reply:
path.resolve returns a string, so there's no difference until the path is correct

Reply:
I know that but I just wanted to be sure if everything's okay. 🔐  If you think it is not necessary we can remove these tests.

Reply:
I think that's ok - we should just check if it can write the file in new nested directories and check if we won't swallow an error.